src/savegame_json.cpp

```diff
@@ -449,6 +449,11 @@ void Character::load( const JsonObject &data )
 
     data.read( "base_age", init_age );
     data.read( "base_height", init_height );
+    int tmp_blood;
+    if( !data.read( "blood_type", tmp_blood ) ) {
+        tmp_blood = rng( 0, 7 );
+    };
+    blood_type = static_cast<blood_types>( tmp_blood );
```

Comment:
Once you have the `io::` stuff working this can be
```suggestion
    if( !data.read( "blood_type", blood_type ) ) {
        blood_type = static_cast<blood_types>( rng( 0, 7 ) );
    };
```